ipc/glue/ProtocolUtils.cpp

```diff
@@ -495,6 +495,9 @@ bool IProtocol::DeallocShmem(Shmem& aMem) {
 
 void IProtocol::SetManager(IProtocol* aManager) {
   MOZ_RELEASE_ASSERT(!mManager || mManager == aManager);
+  recordreplay::RecordReplayAssert("[RUN-916] IProtocol::SetManager %d %d",
```

Comment:
Why cast to `unsigned`, if you take `%d`?

Reply:
Good catch.  I was just making sure varargs were int-sized.  You're right it should be '%u %u', or a cast to `(int)` instead in the args.